packages/registry/test/registry/component-metadata.spec.tsx

```diff
@@ -59,9 +59,17 @@ describe('Component Metadata', () => {
     });
   });
 
-  describe('A11y metadata', () => {
-    it('returns default value equals to false for nonA11yCompliant', () => {
+  describe('Compliancy flags', () => {
```

Comment:
I think the title should be more informative about the flags (only because the file name is not visible in the describe titles)